### PR TITLE
ROOT_EFFECTS_INIT actions as ActionCreators

### PR DESCRIPTION
**Pull request by maxisam**
https://github.com/ngrx/platform/issues/2218

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by maxisam** [comment]
not sure what I did wrong.

File: modules/effects/src/effects_root_module.ts [hunk 1]
```diff
@@ -10,6 +11,7 @@ import { EffectSources } from './effect_sources';
 import { ROOT_EFFECTS, _ROOT_EFFECTS_GUARD } from './tokens';
 
 export const ROOT_EFFECTS_INIT = '@ngrx/effects/init';
+export const rootEffectsInitAction = createAction(ROOT_EFFECTS_INIT);
```
Comment (brandonroberts) [inline modules/effects/src/effects_root_module.ts]:
```suggestion
export const rootEffectsInit = createAction(ROOT_EFFECTS_INIT);
```

File: modules/effects/src/index.ts [hunk 2]
```diff
@@ -8,7 +8,10 @@ export { Actions, ofType } from './actions';
 export { EffectsModule } from './effects_module';
 export { EffectSources } from './effect_sources';
 export { EffectNotification } from './effect_notification';
-export { ROOT_EFFECTS_INIT } from './effects_root_module';
+export {
+  ROOT_EFFECTS_INIT,
+  rootEffectsInitAction,
```
Comment (timdeschryver) [inline modules/effects/src/index.ts]: The action creator is created as `rootEffectsInit ` in `effects_root_module.ts`

```suggestion
  rootEffectsInit,
```

**Comment by maxisam** [comment]
thanks guys!

**Comment by brandonroberts** [comment]
Thank you @maxisam!

**Pull request merged**